js/stateless.js/src/idls/light_system_program.ts

```diff
@@ -1044,23 +1081,36 @@ export const IDL: LightSystemProgram = {
                     name: 'accountCompressionAuthority',
                     isMut: false,
                     isSigner: false,
+                    docs: [
+                        'This pda is used to invoke the account compression program.',
+                    ],
                 },
                 {
                     name: 'accountCompressionProgram',
                     isMut: false,
                     isSigner: false,
+                    docs: ['Merkle trees.'],
```

Comment:
unclear docs

Reply:
This is an error in the idl the full comment reads:
```
/// CHECK: Account compression program is used to update state and address
    /// Merkle trees.
    ```